### ISSUE TITLE
The interest of minibatches

### ISSUE THREAD

**Issue by Hebbalali** [ICL-SML/Doubly-Stochastic-DGP#30]
Hello,

I have a naive question about the use of minibatches. Since the ELBO you derive in your work is a generalization of the one used by Hensman,2013 (the not collapsed formulation) to multiple layers keeping the fully factorized form according to data points. The computation of the ELBO can be parallelised according to the data points.  
Does minibatches allows this parallelization? 

Thank you in advance for your answer

**Comment by hughsalimbeni** [comment]
Yes, you're exactly right. There is a parallelism over the data, so tensorflow is already computing things in parallel within the batch.
